swap_meet/clothing.py

```diff
@@ -0,0 +1,9 @@
+from swap_meet.item import Item
+class Clothing(Item):
+    pass
+    def __init__(self,condition = 0.0):
+        self.category = "Clothing"
+        self.condition = condition
```

Comment:
Another way to write this is to use super to take advantage of the code in Item's constructor:
```
def __init__(self, condition = 0.0):
    super().__init__("Clothing", condition)
```